src/flint/fmpz_mod_mpoly.jl

```diff
@@ -145,6 +145,11 @@ for (etype, rtype, ftype, ctype) in (
                         a, parent(a)))
     end
 
+    function fit!(a::($etype), n::Int)
+      # needs to exist for the MPoly interface
+      return nothing
```

Comment:
If an empty method is a valid default implementation, then why don't we just add `fit!(::MPolyRing) = nothing` to AbstractAlgebra instead of this PR here?

Reply:
from the way that the MPoly interface is formulated in the AA docs, it seemed to me that this function should not have a default implementation. Here, it is fine to do it like this, since Flint does not have the need of this getting called.